app/views/oidc/api_key_roles/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<% @title = t(".api_key_roles") %>
+
+<div class="t-body">
+  <table class="owners__table">
+    <thead>
+      <tr class="owners__row owners__header">
+        <th class="owners__cell">
+          <%= OIDC::ApiKeyRole.human_attribute_name(:name) %>
+        </th>
+        <th class="owners__cell">
+          <%= OIDC::ApiKeyRole.human_attribute_name(:token) %>
+        </th>
+        <th class="owners__cell">
+          <%= OIDC::Provider.human_attribute_name(:issuer) %>
+        </th>
+      </tr>
+    </thead>
+    <tbody class="t-body">
+      <% @api_key_roles.each do |api_key_role| %>
+        <tr class="owners__row">
+          <td class="owners__cell" data-title="Name">
+            <%= link_to api_key_role.name, profile_oidc_api_key_role_path(api_key_role.token) %>
+          </td>
+          <td class="owners__cell " data-title="Role Token">
+            <code><%= api_key_role.token %></code>
+          </td>
+          <td class="owners__cell" data-title="Provider">
+            <%= link_to api_key_role.provider.issuer, api_key_role.provider.issuer %>
```

Comment:
When this is setup on prod, could we name the issuer `Github Actions`? is `provider.issuer` the same as `provider.configuration.issuer`? If so, could we remove `provider.issuer` and add a name attribute and store it as `Github Actions`.

<img width="707" alt="Screenshot 2023-08-14 at 8 16 58 PM" src="https://github.com/rubygems/rubygems.org/assets/42748004/b0a05610-b19f-40c9-9546-86bb31be9fa4">